config/swagger.yaml

```diff
@@ -247,6 +257,43 @@ paths:
           schema:
             $ref: '#/definitions/Error'
       security: []
+
+## CAUTION: debug endpoints, implementation may be inefficient
+  /balances:
+    get:
+      tags:
+        - external
+      operationId: GetAccountsBalances
+      description: 'Get all users'' balances'
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters: []
+      responses:
+        '200':
+          description: successful operation
+          schema:
+            $ref: '#/definitions/AccountsBalances'
+      security: []
+  /info:
+    get:
+      tags:
+        - external
```

Comment:
internal or external?

Reply:
I'd say external. It might be worth dividing into `GET /version` and GET`/debug-info` one day.